src/core/server/integration_tests/saved_objects/migrations/group3/split_failed_to_clone.test.ts

```diff
@@ -139,7 +139,7 @@ describe.skip('when splitting .kibana into multiple indices and one clone fails'
     );
 
     // remove the failure
-    await client.cluster.putSettings({ persistent: { 'cluster.max_shards_per_node': 20 } });
+    await client.cluster.putSettings({ persistent: { 'cluster.max_shards_per_node': 22 } });
```

Comment:
Account for the 2 new indices that need to be created: 
* `.kibana_task_manager_new_<current_version>_reindex_temp`
* `.kibana_task_manager_new_<current_version>_001`